omod/src/main/webapp/resources/scripts/custom/visits.js

```diff
@@ -131,7 +131,7 @@ visit.showEditVisitDateDialog = function(visitId) {
             actions: {
                 confirm: function() {
                     var url = emr.fragmentActionLink("coreapps", "visit/visitDates", "setDuration");
-                    $.getJSON(url, $('#edit-visit-dates-dialog-form-' + visitId).serialize()).success(function(data) {
+                    $.getJSON(url, $('#edit-visit-dates-dialog-form-' + visitId).serialize()).done(function(data) {
                             if (data.success) {
```

Comment:
Thanks @reagan-meant  .Is this the only occuranceof    .succes()  ??

Reply:
@mozzy11 I have changed the error method to fail